tests/import_test.go

```diff
@@ -157,7 +157,7 @@ var _ = Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:compo
 		Expect(importer.DeletionTimestamp).To(BeNil())
 	})
 
-	It("[test_id:6688] Should retain all multi-stage importer pods after completion with dv annotation cdi.kubevirt.io/storage.pod.retainAfterCompletion=true", func() {
+	It("[test_id:6688] Should retain all multi-stage importer pods after completion with vddk dv annotation cdi.kubevirt.io/storage.pod.retainAfterCompletion=true", func() {
```

Comment:
I think something like labels would work a lot better for your use case, and less likely to break:
https://github.com/kubevirt/kubevirt/blob/main/tests/decorators/decorators.go#L21

Reply:
Something we also thought about, but as it only affected a single test, this seemed like a less cluttery solution.
A label like this would be fine?
```suggestion
	It("[test_id:6688] [VDDK] Should retain all multi-stage importer pods after completion with dv annotation cdi.kubevirt.io/storage.pod.retainAfterCompletion=true", func() {
```

Reply:
No, I meant like a real ginkgo label, if you look up the one I linked through the kubevirt test project, you'll see what I mean

Reply:
Ah missed your link, sorry.
Ok so you'd like to introduce labeling mechanism in the same fashion for cdi, as it is not used yet here?

Reply:
I vote for real ginkgo labels & documented, ultimately.  e.g. [arch:s390x], [arch:arm64] with the default being understood to be [arm:amd64] so it's clearer to people maintaining this code what is going on.  

It'd be a more verbose PR later, which would not conflict with this change at all.

Reply:
These cleartext additions have been a pain in kubevirt, they ended up being duplicated in the test name and labels. I would prefer something like Label("VDDK") (don't have to go further than that)

Reply:
Alright
I'll enable filtering in a separate PR then